scripts/rebuild.sh

```diff
@@ -0,0 +1,6 @@
+docker-compose down
```

Comment:
This is a script which I have found very useful in testing, but could be dangerous to run without knowing its purpose. We might consider excluding this file.

Reply:
Removed this script in [b7aac5e](https://github.com/NTIA/scos-sensor/pull/218/commits/b7aac5eef24e809ef7ca12d371bf7d930189addc) since it is potentially dangerous and very easy to re-create locally when needed. It is also not a useful script in production.